app/assets/javascripts/react_on_rails.js

```diff
@@ -65,10 +65,10 @@
     try {
       var reactElement = createReactElement(componentName, propsVarName, props,
         domId, trace, generatorFunction);
-      htmlResult = React.renderToString(reactElement);
+      htmlResult = provideServerReact().renderToString(reactElement);
     }
     catch (e) {
-      htmlResult = handleError(e, componentName);
+      htmlResult = handleError(e, componentName, provideServerReact());
```

Comment:
We're calling this with a 3rd param of the provideServerReact().

Next PR uses an options hash.